File: Core Java/Day2 Assignment/My Own AutoShop Project/assignment/Car.java
package assignment;

public class Car {
	int speed;
	static double RPrice;
	String color;
	
	public Car(){
		
	}
	
	public Car(int a, double b, String c){
		this.speed = a;
		this.RPrice = b;
		this.color = c;
	}
	
	public double getSalePrice(float percent) {
		double gst = percent/100;
		
		double totalgst = gst * RPrice;
		double total = totalgst + RPrice;
		return total;
	}
	public void display() {
		System.out.println("Car Speed: "+speed);
		System.out.println("Car Price : "+RPrice);
		System.out.println("Car Color : "+color);
	}

}

File: TypeScriptExample/Car.ts
class Car{
    Speed:number=12;
    RPrice:number=123;
    Color:string="Red"
    constructor(i:number,n:number,s:string){
        this.Speed=i;
        this.RPrice=n;
        this.Color=s;
    }
    Car(){

    }
    display()
    {
        console.log(this.Speed);
        console.log(this.RPrice);
        console.log(this.Color);
    }
}
function getSalePrice(prdPrice) {
    var totalPrice =(prdPrice/100)+prdPrice*0.18;
    
    return totalPrice
    
    }



console.log(getSalePrice(230));